scripts/opencode-model-scout.sh

```
@@ -0,0 +1,295 @@
+    return 0
+  fi
+
+  printf 'Scout: probing %d candidate model(s) (parallel=%d, heartbeat timeout=%ss)\n' "$total" "$PARALLEL" "$TIMEOUT"
+
+  local launched=0
+  local -a jobs=()
+  while IFS= read -r model; do
+    [[ -z "$model" ]] && continue
+    # setsid: give each probe its own process group so the watchdog can kill the
+    # whole tree (opencode + its children) with a single group kill.
+    setsid bash -c 'probe_model "$1"' _ "$model" &
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)